pyaedt/rpc/rpyc_services.py

```diff
@@ -42,10 +42,11 @@ def on_disconnect(self, connection):
         # code that runs after the connection has already closed
         # (to finalize the service, if needed)
         if self.app:
-            try:
-                self.app[0].release_desktop()
-            except:
-                pass
+            if not os.name == "posix":
+                try:
+                    self.app[0].release_desktop()
+                except:
+                    pass
```

Comment:
Maybe we can raise an exception here to say that the desktop was not released properly.
`raise Exception("The desktop was not released properly.") `

Reply:
I've removed the try except since should be handled by the release destkop